CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ NOTE: [`epaint`](crates/epaint/CHANGELOG.md), [`eframe`](crates/eframe/CHANGELOG
 * MSRV (Minimum Supported Rust Version) is now `1.65.0` ([#2314](https://github.com/emilk/egui/pull/2314)).
 * ⚠️ BREAKING: egui now expects integrations to do all color blending in gamma space ([#2071](https://github.com/emilk/egui/pull/2071)).
 * ⚠️ BREAKING: if you have overlapping interactive widgets, only the top widget (last added) will be interactive ([#2244](https://github.com/emilk/egui/pull/2244)).
+* Improved plot grid appearance ([#2412](https://github.com/emilk/egui/pull/2412)).
```

Comment:
This is still in the wrong place - you need to merge in `main` before moving it!

Reply:
Sorry about that, should be okay in https://github.com/emilk/egui/pull/2412/commits/52ee847dbc7db65ea1e9963460bf99d489308dca.